docs/use-cases/resources.md

```diff
@@ -0,0 +1,124 @@
+
+Further types of App-level resource are on the Convox roadmap to implement.  If you have specific demands, please contact us through the Support page in your console to request them.
+
+For further information see the dedicated page on [App Resources](/application/resources).
+
+
+# Rack-level Resources
+
+Rack-level resources are not supported on your local rack, only in a cloud backed Rack.  They are created and configured from the Convox CLI. The following resources are available at the Rack level:
+
+* `s3`
+* `sns`
+* `sqs`
+* `syslog`
+* `webhook`
+* `memcached` *
+* `mysql` *
+* `postgres` *
+* `redis` *
+
+<div class="block-callout block-show-callout type-info" markdown="1">
+ * Unless required at the Rack level, we would recommend using these types at the App level.
+</div>
+
+
+### Creating a Rack Resource
+
+You can create a Rack resource very simply with the default options or you can pass in your optional values at creation time.
+
+```
+$ convox rack resources create postgres
+Creating resource... OK, postgres-8458
+
+$ convox rack resources create postgres MultiAZ=true
+Creating resource... OK, postgres-2871
+```
+
+### Listing Current Rack Resources
+
+```
+$ convox rack resources
+NAME                  TYPE     STATUS
+console-175092fe6ab1  webhook  running
+syslog-2984           syslog   running
+postgres-8458         postgres running
+postgres-2871         postgres running
+```
+
+#### Options
+
+You can see the available options for a resource type using the CLI:
+
+```
+$ convox rack resources options postgres
+NAME                        DEFAULT      DESCRIPTION
+AllocatedStorage            10           Allocated storage size (GB)
+BackupRetentionPeriod       1            The automatic RDS backup retention period, (default 1 day)
+Database                    app          Default database name
+Encrypted                   false        Encrypt database with KMS
+EngineVersion               9.6          Version of Postgres
+Family                      postgres9.6  Postgres version family
+InstanceType                db.t2.micro  Instance class for database nodes
+MultiAZ                     false        Multiple availability zone
+Password                    (generated)  Server password
+Username                    postgres     Server username
+```
+
+### Updating a Rack Resource
+
+```
+$ convox rack resources update postgres-8458 MultiAZ=true
+```
+
+### Deleting a Rack Resource
+
+```
+$ convox rack resources delete postgres-8458
+Deleting postgres-8458... OK
+```
+
+## Use Cases
```

Comment:
This section should either be greatly elaborated (each resource gets its own h3-section describing it) or removed.